### DISCUSSION TITLE
Make Daily Heroic IDs reset at the same time as Daily Quests

### DISCUSSION THREAD

**Discussion by NinesCC** [chromiecraft/chromiecraft#5871] [Suggestions]
Daily ID resets at 4:00 UTC
Daily Quests reset at 6:00 UTC

This means that there are 2 dead hours when people will avoid queuing in order to not get saved for the Daily Heroic quest. This is detrimental to NA queues in general, as heroics stop popping too early.

I would recommend just moving the Daily ID reset to the same time as the Daily Quest or the other way around, but having them reset at different times makes it awkward and doesn't help anyone.

**Comment by balleny** [comment]
please use the search function before posting duplicates
https://github.com/chromiecraft/chromiecraft/discussions/5751

**Reply by NinesCC** [reply]
Ooops, my bad! Weird that it hasn't been given any attention then.
